### ISSUE TITLE
[Accessibility]A11y_VSCode_ActivityBar_Keyboard: Keyboard trap occurred on "ActivityBar" item list except "Setting" button.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52006]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : MAC OS HighSierra10.13.5

**Additional Details:**
Keyboard,MAS2.1.2,MAS2.4.3

**Repro Steps:**
1.Navigate to "Activity Bar" at left side of the screen after login.
3.Navigate through keyboard such as "Tab" key.
4.Observed that, Keyboard trap is occured between "Explorer,Search,SouceControl,Debug,Extensions" except "settings" option.

**Actual:**
While navigating through the key, the keyboard trap is occured between "Explorer,Search,SouceControl,Debug,Extensions"except "settings" option.

**Expected:**
Without keyboard trap each control should be accessible by keyboard, here keyboard focus should be goes on each control. 

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx](url)

**User Impact:**
User get trapped in some actionable controls of ActivityBar.

**MAS Reference:**
MAS2.1.1- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={ebdc3fe1-09a9-4aa2-a29b-e97764774e6c}](url)

MAS2.4.3- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9}](url)

**Attachment For Reference:**
[[Accessibility]A11y_VSCode_ActivityBar_Keyboard.pptx](https://github.com/Microsoft/vscode/files/2105572/Accessibility.A11y_VSCode_ActivityBar_Keyboard.pptx)
Does this issue occur when all extensions are disabled?: Yes

**Comment by sandy081** [comment]
@AccessibilityTestingTeam-TCS I could access the gear icon through tab in the activity bar. 

Let me know what am I missing here

**Comment by AccessibilityTestingTeam-TCS** [comment]
The issue is mentioned in attached ppt. This issue occured in MAC, when user is on activity bar and move to next item of activity bar the using tab user can navigate upto "Extensions" button only. Here keyboard trap is occured and user is not able to move to "Setting" option.Here without keyboard trapp all the control should be accessible by the keys.
